### PR TITLE
Chmod script files and use name of benchmark for .bc file.

### PR DESCRIPTION
**Pull request by patrickbird**

### CODE REVIEW

File: runscripts/run-section2.sh [hunk 1]
```diff
@@ -8,24 +8,23 @@ LDFLAGS=
...
 
-FILES=$1/*.bc
-for f in $FILES
-do
-	filename=$(basename "$f")
-	filename="${filename%.*}"
-	opt -load $LLVMLIB/CSE231.so -dynamic < $f > $CSE231ROOT/$filename.pass.bc
-	## compile the instrumentation module to bitcode
-	clang $CPPFLAGS -O0 -emit-llvm -c $INSTRUMENTATION/dynamic/CountDynamicInstructionsInstrumentation.cpp -o $INSTRUMENTATION/dynamic/CountDynamicInstructionsInstrumentation.bc
+set -x
```
Comment (patrickbird) [inline runscripts/run-section2.sh]: Whoops...left `set -x` there.

**Comment by patrickbird** [comment]
So most of the errors go away, but this is the only one that remains:

```
diff: out.gold.dat: No such file or directory    
```

Looking into it now.  This is only for hadamard.

File: runscripts/run-section2.sh [hunk 2]
```diff
@@ -8,24 +8,23 @@ LDFLAGS=
...
+
+filename=$(basename "$1")
+filename="${filename%.*}"
```
Comment (scheah) [inline runscripts/run-section2.sh]: line 14 not needed

File: runscripts/run-section3.sh [hunk 3]
```diff
@@ -8,24 +8,20 @@ LDFLAGS=
...
-FILES=$1/*.bc
-for f in $FILES
-do
-	filename=$(basename "$f")
-	filename="${filename%.*}"
-	opt -load $LLVMLIB/CSE231.so -branchbias < $f > $CSE231ROOT/$filename.pass.bc
-	## compile the instrumentation module to bitcode
-	clang $CPPFLAGS -O0 -emit-llvm -c $INSTRUMENTATION/branchbias/BranchBiasInstrumentation.cpp -o $INSTRUMENTATION/branchbias/BranchBiasInstrumentation.bc
+filename=$(basename "$1")
+filename="${filename%.*}"
```
Comment (scheah) [inline runscripts/run-section3.sh]: line 12 not needed

**Comment by scheah** [comment]
we should replace $CSE231ROOT with $1

Shouldn't dump our transformed btye code in the root folder.

**Comment by scheah** [comment]
Final bc file is the same as the directory name. This needs to be reflected for section 1 script also

**Pull request merged**